glamorous/button.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import glamorous from 'glamorous';
+
+const MyStyledButton = glamorous.button({
+  backgroundColor: '#ff0000',
+  width: 320,
+  padding: 20,
+  borderRadius: 5,
+  border: 'none',
+  outline: 'none',
+  ':hover': {
+    color: '#fff',
+  },
+  ':active': {
+    position: 'relative',
+    top: 2,
+  },
+  '@media(max-width: 480px)': {
+    width: 160,
+  },
+});
+
+const Button = () => (
+  <glamorous.Div textAlign="center">
```

Comment:
I actually had that and reverse it. I will use the deconstructed and then add a comment about using it through namespace.